### DISCUSSION TITLE
It is no longer possible to download packages from the GitHub UI

### DISCUSSION THREAD

**Discussion by Th3S4mur41** [community/community#69687] [Packages]
### Select Topic Area

Bug

### Body

It the past, the asset bound to a package (e.g. npm) was provided in the GitHub UI as a link that made it downloadable from the GUI.
![image](https://github.com/community/community/assets/5543595/00cb6d52-c32d-4387-822b-b6dd82f4df38)

This is no longer the case, so the feature is either broken or removed.
![image](https://github.com/community/community/assets/5543595/f410a449-0957-467b-82b2-3212384a7584)

This was pretty useful and should definitely be added back in.

See also https://stackoverflow.com/a/69729793/7041074

**Comment by esurov** [comment]
Dear @Th3S4mur41 ,

As of my last knowledge, GitHub did allow you to attach binary assets to releases, and these assets could be downloaded directly from the GitHub UI. This was especially useful for distributing packages like npm, as you mentioned. However, GitHub's features and user interface may have changed since then.

If you're experiencing issues with this feature or believe it has been removed, it's possible that GitHub has made updates to their platform or there might be a configuration issue with your specific repository. To resolve this, consider the following steps:

1. **Check GitHub's Documentation**: Review the official GitHub documentation and release notes to see if there have been any changes to the way binary assets are handled.

2. **Repository Settings**: Ensure that your repository settings allow for binary assets to be attached to releases. If the repository is set to "private," you might need to check whether this feature is restricted in any way.

3. **Browser and Extensions**: Sometimes browser extensions can interfere with web functionality. Try disabling any extensions that might be causing issues with the GitHub UI.

4. **Contact GitHub Support**: If you're unable to resolve the issue, consider reaching out to GitHub support for assistance. They can provide guidance and help troubleshoot any issues you're experiencing.

5. **Use the GitHub API**: If downloading assets from the UI remains a challenge, you can consider using the GitHub API to programmatically access and download assets associated with releases.

6. **Community Feedback**: You can also share your feedback with GitHub. They actively listen to user feedback and may consider re-implementing features based on the community's needs and requests.

Please note that the state of software and web platforms can change rapidly, and the information I provided is based on my last update in January 2022. It's possible that GitHub has made updates or changes to their platform since then. I recommend checking the latest GitHub documentation and community forums for the most up-to-date information on this matter.

**Comment by TheKnarf** [comment]
This feature is still missing...

**Comment by maximetinu** [comment]
I also need this

**Comment by maltfield** [comment]
If you want to download a file from GitHub Packages, you can do it via the API and `curl`.

It took me a few weeks to figure out how to do this, so I wrote an article describing it [here](https://tech.michaelaltfield.net/2024/09/03/container-download-curl-wget/)

 * https://tech.michaelaltfield.net/2024/09/03/container-download-curl-wget/

**Reply by TheKnarf** [reply]
That will only work for Docker containers as far as I understand, not for `npm` packages.

**Reply by maltfield** [reply]
The article has two examples: one is docker and the second is GitHub Packages. Specifically read the [section about GitHub Packages](https://tech.michaelaltfield.net/2024/09/03/container-download-curl-wget/#github-packages) -- it should apply to `npm` packages.

 * https://tech.michaelaltfield.net/2024/09/03/container-download-curl-wget/#github-packages

**Comment by jpshack-at-palomar** [comment]
Also missing this feature.

**Comment by victor9000** [comment]
Please fix
